src/schemas/scriptExecutions/ActionExecutionDto.yaml

```diff
@@ -0,0 +1,35 @@
+type: object
+properties:
+  runId:
+    type: string
+  processId:
+    type: integer
+    format: int64
+  type:
+    type: string
+  name: 
+    type: string
+  description:
+    type: string
+  condition:  
+    type: string
+  errorStop:
+    type: boolean
+  errorExpected:
+    type: boolean
+  status:
+    type: string
+    enum: ["RUNNING", "SUCCESS", "WARNING", "ERROR", "STOPPED", "SKIPPED"]
+  startTimestamp:
+    type: string
+  endTimestamp:
+    type: string
```

Comment:
Can you mark these as 'timestamps'?

Reply:
Do you mean I have to change the format into "date-time" ? The variable names are the same as the class defined in the rest-server

Reply:
Indeed, you just mark the the format as 'date-time' (https://swagger.io/docs/specification/data-models/data-types/#string)

Reply:
Got it